dart/lib/src/sentry_stack_trace_factory.dart

```diff
@@ -103,6 +103,7 @@ class SentryStackTraceFactory {
       // least we get an indication something's wrong and are able to fix it.
     }
 
+    final platform = _options.platformChecker.isWeb ? 'javascript' : 'dart';
```

Comment:
does this mean we aren't going to see the Dart stack traces coming from the web platform?

Reply:
this is just about the raw formatting, you will still be able to see everything as before

Reply:
before it wasn't showing proper Dart-formatted stack traces on any platform.

Need to be able to see the dart stacktraces on desktop, mobile and web targets.

Also, this seem to be SDK change? The correct stack traces won't be shown on Sentry's web UI until the apps are updated and new errors/stacktraces are pushed?

Reply:
see my comment  here about what we can do if you want it to work on existing issues:

https://github.com/getsentry/sentry-dart/issues/2040#issuecomment-2250159494

let's continue the discussion over there